WooCommercePOS/WooCommercePOS/Presentation/PointOfSaleEntryPointView.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+public struct PointOfSaleEntryPointView: View {
+    // TODO:
+    // Temporary. DI proper product models once we have a data layer
+    private let viewModel: PointOfSaleDashboardViewModel = {
+        let products = ProductFactory.makeFakeProducts()
+        let viewModel = PointOfSaleDashboardViewModel(products: products)
+
+        return viewModel
+    }()
+
+    private var shouldHideWooAppTabBar: ((Bool) -> Void)? = nil
+
+    // Necessary to expose the View's entry point to WooCommerce
+    // Otherwise the current switch on `HubMenu` where this View is created
+    // will error with "No exact matches in reference to static method 'buildExpression'"
+    public init(hideAppTabBarsCallback: ((Bool) -> Void)? = nil) {
```

Comment:
nit: as the value type already implies a closure/callback, it can be omitted in the name

```suggestion
    public init(hideAppTabBar: ((Bool) -> Void)? = nil) {
```

Reply:
Done between https://github.com/woocommerce/woocommerce-ios/commit/27f8499b3be1c89df547ff94f656e9804309d9fc and https://github.com/woocommerce/woocommerce-ios/pull/12700/commits/a0cd382f21f345f7becfa15b011fb960e20c693c